src/cli/Microsoft.DotNet.UpgradeAssistant.Cli/Commands/AnalyzeBinaries/ConsoleAnalyzeBinariesCommand.cs

```diff
@@ -27,14 +25,48 @@ public ConsoleAnalyzeBinariesCommand()
                     .UseConsoleUpgradeAssistant<ConsoleAnalyzeBinaries>(options, result)
                     .RunUpgradeAssistantAsync(token));
 
-            AddArgument(new Argument<FileSystemInfo[]>("files-or-directories", LocalizedStrings.BinaryAnalysisContentHelp)
+            AddArgument(new Argument<FileSystemInfo[]>("files-or-directories", parse: r =>
+            {
+                if (r?.Tokens is null || !r.Tokens.Any())
+                {
+                    throw new ArgumentException(@"Must specify target file/directory for analysis");
+                }
+
+                return r.Tokens.Select<Token, FileSystemInfo>(i =>
```

Comment:
This may be nice to move into a helper method. I imagine this kind of thing to be reused potentially

Reply:
I'm not supposed to have to do it at all; filed a bug on their repo for it. So hopefully it's deleteable in the near future.

Reply:
Can you link that here?